### PR TITLE
Add azimuth-cli project

### PR DESCRIPTION
**Pull request by mikeosborne**

### CODE REVIEW

**Comment by lukebuehler** [comment]
Thank you, Mike. Reviewing right now. Will add anything that is still missing.

**Comment by lukebuehler** [comment]
Added #1149

File: content/grants/azimuth-cli.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+## Azimuth CLI
+
+The azimuth-cli is "Bridge for the command line." It should allow ship owners and operators to do the same things that they can in Bridge but via the command line, and in batch mode. For example, it should be possible to batch-spawn 10 planets from a star, or transfer ownership of several stars or planets with one command.
+
+## Background
+
+Azimuth is a set of Ethereum smart contracts that handle most things related to Urbit ID. The primary way of interacting with Azimuth, right now, is via Bridge, a dApp. However, hosting providers and people who run stars frequently need to automate their interactions with Azimuth, but there is currently not an easy way to do that.
+
+There is the excellent azimuth-js library that makes most bridge functions available as a JavaScript API, but it still requires substantial effort to automate things. Additionally, not all the features available in Bridge are currently accessible via npm package, requiring quite a bit of reverse engineering.
```
Comment (Fang-) [inline content/grants/azimuth-cli.md:26]: > not all the features available in Bridge are currently accessible via npm package

> 2. Improve the current Urbit ID related JS projects, such as azimuth-js, and urbit-key-generation.

Let's please not add this to azimuth-js, which is intended to just be a shallow layer around the contracts themselves. Things like "send a planet invite" should probably go in their own library. (Is there even any functionality "missing" beyond that?)

Comment (lukebuehler) [reply content/grants/azimuth-cli.md:26]: I agree, no real changes needed in azimuth-js. I think where I can contribute is urbit-key-generation, e.g., arvo keyfile gen, and reverse from keyfile to private/public keys.

**Review by jalehman — APPROVED** [review]

**Pull request merged**